test/statsCases/tree-shaking/expected.txt

```diff
@@ -1,7 +1,7 @@
-Hash: e94a2c6bee98efb02ae8
+Hash: 00d5541377fcfa0231b5
 Time: Xms
-    Asset     Size  Chunks             Chunk Names
-bundle.js  7.14 kB       0  [emitted]  main
+    Asset    Size  Chunks             Chunk Names
+bundle.js  7.6 kB       0  [emitted]  main
```

Comment:
Unminified bundle should grow, possibly quite a bit if there are many small harmony modules, but they _should_ minify to the same size.